.tool-versions

```diff
@@ -1 +1 @@
-scarb 2.4.0
+scarb nightly-2023-12-18
```

Comment:
I assume we merge this one when 2.5.0 is released?

Reply:
If not, we shouldn't merge this change to `.tool-versions`

Reply:
Why? setup scarb action detects .tool-versions and takes scarb version from it (yea, I got to know that yesterday, didn't expect it too xD)

Reply:
Because it is unstable and we shouldn't run our tests on it.

Reply:
If we want to merge this one we have to run tests on this version. The other option is waiting with merging this PR until new stable scarb comes out

Reply:
>If we want to merge this one we have to run tests on this version.

we should either ignore those tests or wait, we should not compromise existing features because of the new ones.